packages/gateway/src/features/metrics/root-resolvers.ts

```diff
@@ -59,6 +59,22 @@ export const resolvers: GQLResolver = {
         authHeader
       )
     },
+    async getTotalCorrections(
+      _,
+      { timeStart, timeEnd, locationId, event },
+      authHeader
+    ) {
+      return getMetrics(
+        '/totalCorrections',
+        {
+          timeStart,
+          timeEnd,
+          locationId,
```

Comment:
@rikukissa a small question, even if we don't pass `locationId` from client, will it cause metrics to get a string 'undefined' ?